.github/actions/release-image-terraform/action.yml

```diff
@@ -23,99 +23,34 @@ runs:
         repository: ${{ inputs.overrideCheckoutRepository }}
 
     # optionally fetch a gcs bucket to be used by melange and apko builds
-    - id: gcsfetchauth1
-      if: inputs.gcsFetchBucketName != ''
+    - if: inputs.gcsFetchBucketName != ''
       uses: google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d # v1.0.0
       with:
         workload_identity_provider: ${{ inputs.gcsFetchAuthWorkloadIdentityProvider }}
         service_account: ${{ inputs.gcsFetchAuthServiceAccount }}
-    - id: gcsfetchauth2
-      if: inputs.gcsFetchBucketName != ''
+    - if: inputs.gcsFetchBucketName != ''
       uses: google-github-actions/setup-gcloud@d51b5346f85640ec2aa2fa057354d2b82c2fcbce # v1.0.1
       with:
         project_id: ${{ inputs.gcsFetchAuthProjectId }}
-    - id: gcsfetchauth3
-      if: inputs.gcsFetchBucketName != ''
+    - if: inputs.gcsFetchBucketName != ''
       shell: bash
       name: 'Check that GCloud is properly configured'
       run: |
         gcloud info
         gcloud --quiet alpha storage ls
-    - id: rsync
-      shell: bash
+    - shell: bash
```

Comment:
You can drop these too

Reply:
I think we need these in `.github/actions` 🤔